pkg/extract/extract.go

```diff
@@ -117,9 +119,18 @@ func untarFile(tarReader *tar.Reader, header *tar.Header, path string) error {
 	}
 	defer file.Close()
 
+	var reader io.Reader
+	reader = tarReader
```

Comment:
This is only implemented for tar files? (rationale in commit log)

Reply:
I added support for zip file also.